packages/light.js-react/package.json

```diff
@@ -38,7 +38,8 @@
     "enzyme-adapter-react-16": "^1.2.0",
     "jest-enzyme": "^6.0.3",
     "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "react-dom": "^16.4.2",
+    "typescript": "^3.1.6"
```

Comment:
devDep

Reply:
It doesn't look like any change is required because `"typescript": "^3.1.6"` is already within `"devDependencies": {`

Reply:
<img width="252" alt="screenshot 2019-01-11 at 10 04 21" src="https://user-images.githubusercontent.com/1293565/51024068-d7b1cc80-1588-11e9-9d6e-61efcf95525e.png">

My bad! I read a bit too quickly